Update README with sl-jupyterhub breaking changes (#103)
ni/install-systemlink-enterprise: release-notes/2022-12/README.md

- `notification 0.1.0`
    - Newly included chart
    - You must configure a secret for MongoDB credentials in `notification.secrets.mongodb`

> - `sl-jupyterhub 0.2.0`

Reviewer: Should this value be added to the [getting started template](https://github.com/ni/install-systemlink-enterprise/blob/main/getting-started/templates/systemlink-values.yaml) with a comment and perhaps an ATTENTION tag so that the user's attention is called to this choice.

Author: updated

>     - The default configuration doesn't allow anymore user pods to establish outbound connections to private IPs (for example accessing SystemLink APIs that are exposed on a private IP from Jupyter Notebooks or the Jupyter terminal). To allow this, you must explicitly set `jupyterhub.singleuser.networkPolicy.egressAllowRules.privateIPs=true`.

Reviewer 2: ```suggestion
    - JupyterHub user pods may no longer establish outbound connections to private IP addresses by default. To enable this, you must explicitly set `jupyterhub.singleuser.networkPolicy.egressAllowRules.privateIPs=true`.
```

Author: I applied @MelissaHilliard 's suggestion. Let me know if you think the text should still be modified

>     - The default configuration doesn't allow anymore user pods to establish outbound connections to private IPs (for example accessing SystemLink APIs that are exposed on a private IP from Jupyter Notebooks or the Jupyter terminal). To allow this, you must explicitly set `jupyterhub.singleuser.networkPolicy.egressAllowRules.privateIPs=true`.

Reviewer 3: ```suggestion
    - To enable JupyterHub user pods to establish outbound connections to private IP addresses by default, set `jupyterhub.singleuser.networkPolicy.egressAllowRules.privateIPs=true`.
```
